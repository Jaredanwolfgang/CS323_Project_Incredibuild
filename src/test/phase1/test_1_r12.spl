int test_1_o02()
{
  int dec_int = 323;
  int hex_int = 0xFFF;
  char ch = '\u6C';
  return hex_int - dec_int;
}
