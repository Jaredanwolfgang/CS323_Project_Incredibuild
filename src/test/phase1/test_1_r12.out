Functions: Function: test_1_o02:[Body: [Variable Declaration: dec_int = [0: u32] with dimensions []; Variable Assignment: dec_int = 323: u32, Variable Declaration: hex_int = [0: u32] with dimensions []; Variable Assignment: hex_int = 4095: u32, Variable Declaration: ch = [ : char] with dimensions []; Variable Assignment: ch = l: char, Return: (hex_int - dec_int)]]